src/main.rs

```diff
@@ -140,7 +136,7 @@ fn main() {
                                .takes_value(true))
 
                           .arg(Arg::with_name("service_job_number")
```

Comment:
We could mark the argument as deprecated and add a new `service_job_id` argument (asserting that only one is used).